wordpress-mobile/WordPress-iOS
# Removes Optimizely. Defaults to old sign in flow.

This PR removes (with prejudice) the Optimizely pod. There have been performance issues that we've been unable to resolve so we'll just remove the pod and look at using Mixpanel for future A/B tests.  The code has been updated to always default to the old sign in flow. 

To test:
Try the sign in flow from multiple avenues (clean start, sign out / back in, self-hosted then wpcom, etc.)
Confirm there are no issues and you always see the old sign in forms. 

Needs review: @sendhil 

cc @kwonye
